docusaurus/docs/cloud/cli/cloud-cli.md

```diff
@@ -62,6 +67,9 @@ Once you deployed your project, if you visit the Strapi Cloud dashboard, you may
 
 Links project in current folder to an existing project in Strapi Cloud.
 
+:::note
+Linking a project to Strapi Cloud doesn't limit it to Strapi Cloud alone; you can still deploy and manage it in your own self-hosted environment as needed.
+
```

Comment:
```suggestion
:::
```

I guess this is where the note callout stops? 😊

Reply:
Yes. I missed that one.